compose/ui/ui/src/desktopMain/kotlin/androidx/compose/ui/awt/ComposeWindowDelegate.desktop.kt

```diff
@@ -58,9 +58,10 @@ internal class ComposeWindowDelegate(
     // so we nullify layer on dispose, to prevent keeping
     // big objects in memory (like the whole LayoutNode tree of the window)
     private var _layer: ComposeLayer? = ComposeLayer(skiaLayerAnalytics)
-    private val layer get() = requireNotNull(_layer) {
-        "ComposeLayer is disposed"
-    }
+    internal val layer
```

Comment:
Maybe instead of making `layer` internal, add
```
internal val windowAccessible: Accessible
    get() = layer.sceneAccessible
```
Not really sure, just an idea.